scrips/calendarVars.js

```diff
@@ -2,5 +2,18 @@ const weekDaysNumber = 7;
 const hoursNumber = 24;
 const cellHeight = 49;
 const currentEventTileId = "current-event-tile";
+const defaultEventLength = 0.75; //hours
+const emptyEventTitle = "(no title)";
+const minutesGap = 15;
```

Comment:
Same thing applies here. What is measurement unit for gaps? Is it pixels?

```suggestion
const minutesGapInPx = 15;
```